warrior/Framework/Utils/data_Utils.py

```diff
@@ -1339,7 +1339,7 @@ def get_filepath_from_system(datafile, system_name, *args):
 
 def get_var_by_string_prefix(string):
     if "ENV" in string:
```

Comment:
Modified condition to check if the string starts with 'ENV', remaining will work fine.